index.js

```diff
@@ -171,27 +172,27 @@ module.exports = function (archive, target, opts, cb) {
     if (dryRun || entry) {
       next()
     } else {
-      next()
-      // TODO: not doing this in v8?
-      // archive.append({
-      //   name: hyperPath,
-      //   type: 'directory'
-      // }, next)
+      archive.mkdir(hyperPath, next)
     }
   }
 
   function next () {
-    consume(target, null, cb)
+    consume(target, null, done)
   }
 
   if (opts.resume) {
-    archive.history({ live: false })
-    .on('error', cb)
-    .on('data', function (entry) {
+    archive.history()
```

Comment:
```
2:50 PM <pfrazee> jhand: hmm the history() call for resume *did* populate the fileCount and totalSize status
2:51 PM <jhand> pfrazee: ah right. 
2:51 PM <pfrazee> jhand: is there a list() method still that just gives the current full listing?
2:51 PM <pfrazee> doesnt seem like it
2:51 PM <jhand> pfrazee: don't think so
2:52 PM <pfrazee> jhand: hmmmm that's tricky because a recursive readdir + stat on each entry cant be super fast either
```